sns/variables.tf

```diff
@@ -24,3 +24,7 @@ variable "sns_policy" {
   description = "allows a custom SNS policy to be set"
   default     = "default"
 }
+
+variable "kms_key_arn" {
+  default = ""
+}
```

Comment:
There is an SNS topic for AWS Config that we're not sure whether we're going to encrypt it yet so this default needs to be here. If we do end up encrypting that topic, then we can remove this.